GOVERNANCE.md

```diff
@@ -185,6 +190,7 @@ Contributors don't need to be part of any dedicated team.
 * Bill Engels (dev)
 * Brandon T. Willard (dev)
 * Chris Fonnesbeck (dev, docs)
+* Christian Luhmann (discourse)
```

Comment:
Should this be community?

Reply:
Im sorry for missing earlier pr, if its not a hassle i propose changing discourse to community. If it is dont worry about it

Reply:
There was no earlier PR on community/discourse team. I sent some messages on slack. I don't have strong opinions on the name. I think the most important thing is to write down the actual goals of the team in the team description section. I got the impression that it was mostly focused on Discourse, not really on any and all community building activities (i.e. sprints or pymcon even) so I started with discourse name and discourse only goals.

Reply:
SGTM its easy enough to swap later. Dont want to hold this up.